style.css

```diff
@@ -116,5 +126,197 @@ h2 {
 }
 
 .social-media a :hover {
-  fill: #3909c8;
+  color: #3909c8;
+}
+
+.section-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+  grid-gap: 88px;
+  padding: 114px 24px;
+}
+
+.project-card {
+  color: #344563;
+  max-width: 327px;
+  border-radius: 16px;
+  padding: 16px;
+  margin: auto;
+  border: solid 1px #8e898d;
+}
+
+.snapshot-portfolio-img {
+  width: 100%;
+  left: 16px;
+  top: 16px;
+  border-radius: 8px;
+}
+
+.left-block {
+  margin-top: 1rem;
+}
+
+.project-title {
+  font-size: 32px;
+  font-weight: 700;
+  margin-bottom: 1rem;
+  font-family: poppins, sans-serif;
+}
+
+.counter {
+  padding: 0;
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+
+.canopy {
+  font-family: poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.canopy-text {
+  display: flex;
+  padding: 0;
+  justify-content: left;
+  gap: 1.25rem;
+  line-height: 24px;
+  letter-spacing: 0.05em;
+  list-style-type: none;
+}
+
+.programming-languages {
+  color: solid #000;
+  display: flex;
+  list-style-type: none;
+  align-items: center;
+  gap: 12px;
+  padding: 0;
+}
+
+.languages a {
+  border-radius: 8px;
+  padding: 4px 12px;
+  background-color: #ebebff;
+}
+
+.about-myself-section {
+  color: #344563;
+  max-width: 375px;
+  border-radius: 16px;
+  padding: 16px;
+  border: solid 1px #8e898d;
+  margin: auto;
+  gap: 12px;
+}
+
+.text-see-project {
+  margin: 0.75rem 0;
+}
+
+.see-project-button {
+  padding: 12px;
+  border: solid 1px #396df2;
+  border-radius: 8px;
+  color: #396df2;
+}
+
```

Comment:
- [ ] Please make that the  `See project`  `button` follows the button styling: Active, Hover, and Pressed (see design)


  ![Screenshot from 2022-12-03 19-12-27](https://user-images.githubusercontent.com/92303891/205457943-227f0990-7771-43ac-9c9c-dc7f6cdd5c46.png)